src/index.html

```diff
@@ -81,7 +85,8 @@ <h1 class="homepage__hero__text">Publish <span class="color-white">mobile, web,
                 <h3>Fast Development</h3>
 
                 <p>
-                Paint your app to life in milliseconds with stateful Hot Reload. Use a rich set of fully-customizable
+                Paint your app to life in milliseconds with stateful
+                Hot Reload. Use a rich set of fully-customizable
```

Comment:
`with stateful Hot Reload`

Our [hot reload doc](https://flutter.dev/docs/development/tools/hot-reload) uses all lowercase i.e. `stateful hot reload`. Should we pick one and go with it?

Reply:
Actually, if you look at go/dart-terms, it's either hot reload (lower case) or Stateful Hot Reload, according to that doc. I believe that came from marketing or the PMs.